lab02-kaylee/test/call-apply-test.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const assert = require('assert');
+const callApply = require('../lib/call-apply.js');
+let newCoffeeDrink = new callApply.CoffeeDrink('light', 'Argentina');
+let newDog = new callApply.Dog('Fido', 'dauchsand', 'brown');
+let newDogArray = newDog.updateBio([1, 2, 3]);
+
+describe('call-apply.js', function() {
+  describe('CoffeeDrink', function() {
+    it('should return an object', function() {
+      console.log('typeof newCoffeeDrink: ', typeof newCoffeeDrink);
```

Comment:
No points lost, but ``console.logs`` in test can also be taken out prior to submission (so THIS would be the case where ``console.logs`` should be removed.)